handler/openid/flow_implicit.go

```diff
@@ -51,6 +51,10 @@ func (c *OpenIDConnectImplicitHandler) HandleAuthorizeEndpointRequest(ctx contex
 		return nil
 	}
 
+	if ar.GetResponseMode() == fosite.ResponseModeNone {
```

Comment:
I think this should not run if `fosite.ResponseModeQuery` and maybe also not with the `FormPost` one. Please confirm @mitar . Also use `ar.SetDefaultResponseMode`.

Reply:
Added `ar.SetDefaultResponseMode`. Using `query` in this flow is already restricted but do we want to restrict `form_post`?